flash/image/classification/cli.py

```diff
@@ -19,9 +19,14 @@
 __all__ = ["image_classification"]
 
 
+# TODO: Find a better way to prevent double injection of input.
 def from_hymenoptera(
     batch_size: int = 4,
     num_workers: int = 0,
+    train_input=None,  # Hack
+    val_input=None,
+    test_input=None,
+    predict_input=None,
```

Comment:
I think this is caused by inspecting args from the old `DataModule` instead of the new one. Should be easy to implement a fix. I'll take a look :smiley:

Reply:
Fixed :smiley: